src/mach/mod.rs

```diff
@@ -313,16 +322,65 @@ impl<'a> Iterator for FatArchIterator<'a> {
     }
 }
 
-/// Iterator over every `MachO` binary contained in this `MultiArch` container
-pub struct MachOIterator<'a> {
+/// Iterator over every entry contained in this `MultiArch` container
+pub struct SingelArchIterator<'a> {
```

Comment:
this is mispelled, should be SIngle not Singel

Reply:
🤦 good catch, thanks